android/src/com/frostwire/android/gui/adapters/TransferListAdapter.java

```diff
@@ -318,9 +318,10 @@ private String populateBittorrentDownloadMenuActions(BittorrentDownload tag, Lis
             } else {
                 boolean wifiIsUp = NetworkManager.instance().isDataWIFIUp();
                 boolean bittorrentOnMobileData = ConfigurationManager.instance().getBoolean(Constants.PREF_KEY_NETWORK_USE_MOBILE_DATA);
+                boolean bittorrentOnlyWifi = ConfigurationManager.instance().getBoolean(Constants.PREF_KEY_NETWORK_USE_WIFI_ONLY);
```

Comment:
wipe this out
bittorrentOnlyWifi == !bittorrentOnMobileData

it is redundant here, will just add headaches.